azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+jobs:
+- job: Linux 
+  strategy:
+    matrix:
+      Node.js 0.10:
+        NODE_VERSION: '0.10'
+      Node.js 0.12:
+        NODE_VERSION: '0.12'
+      Node.js 1.8:
+        NODE_VERSION: '1.8'
+      Node.js 2.5:
+        NODE_VERSION: '2.5'
+      Node.js 3.3:
+        NODE_VERSION: '3.3'
+      Node.js 4.9:
+        NODE_VERSION: '4.9'
+      Node.js 5.12:
+        NODE_VERSION: '5.12'
+      Node.js 6.14:
+        NODE_VERSION: '6.14'
+      Node.js 7.10:
+        NODE_VERSION: '7.10'
+      Node.js 8.12:
+        NODE_VERSION: '8.12'
+      Node.js 9:
+        NODE_VERSION: '9'
```

Comment:
Is this downloading the nightly version like we currently do or is it now downloaded the last released version?

Reply:
No, it is not downloading the nightly version, it is downloading the latest released version. I could add that to my script which does the install.

The Node Tool task (mentioned earlier) also doesn't support nightly versions at this time.